src/components/modals/OrderConfirmationModal.vue

```diff
@@ -1,185 +1,25 @@
 <template>
-  <section class="aph-order-confirmation-modal">
-    <div class="wrapper">
-      <h2 class="title">
-        Confirm Your Order:
-      </h2>
-      <div class="content">
-        <p class="description">
-          Are you sure that you would like to place a
```

Comment:
What happened to all this logic for a nice confirmation modal? 
We cannot replace all this with just a simple "Are you sure you'd like to submit this order?"
We have to show the fees for sure and we need to show what trade is being made. What offers are being taken could be something they have to click to bring up a list or something though.

Reply:
Fixed